eyeshade/workers/reports.js

```diff
@@ -65,10 +68,37 @@ async function freezeOldSurveyors (debug, runtime, olderThanDays) {
   } = await runtime.postgres.query(virtualQuery)
 
   const toFreeze = nonVirtualSurveyors.concat(virtualSurveyors)
-  await Promise.all(toFreeze.map(async (row) => {
-    const surveyorId = row.id
-    await runtime.queue.send(debug, 'surveyor-frozen-report', { surveyorId, mix: true, shouldUpdateBalances: true })
-  }))
+  for (let i = 0; i < toFreeze.length; i += 1) {
+    const surveyorId = toFreeze[i].id
+    await runtime.queue.send(debug, 'surveyor-frozen-report', { surveyorId, mix: true })
+    await waitForTransacted(runtime, surveyorId)
```

Comment:
about how many surveyors are we expecting to be transacted?

Reply:
currently it's about 15/day